src/bin/latgen-faster-mapped-combine.cc

```diff
@@ -0,0 +1,179 @@
+// bin/latgen-faster-mapped.cc
+
+// Copyright 2009-2012  Microsoft Corporation, Karel Vesely
+//                2013  Johns Hopkins University (author: Daniel Povey)
```

Comment:
This is OK for testing but eventually this should just be a change to lattice-faster-decoder.cc.  It's just a small optimization.